Resources/public/templates/confirmbox/draftconflict.hbt

```diff
@@ -0,0 +1,54 @@
+<div class="ez-draft-conflict-header">
+    <div class="ez-draft-conflict-header-subtitle">
+        {{ lookup content.currentVersion.names languageCode }}
+        ({{ translate_property contentType.names }})
+    </div>
+
+    <div class="ez-draft-conflict-header-button">
+        <a class="ez-draft-conflict-link pure-button ez-button"
+            href="{{ path 'editContent'
+                    id=content.id
+                    languageCode=languageCode
+                }}">
+            + New Draft
+        </a>
+    </div>
+</div>
+
+<table class="ez-draft-conflict-table">
+    <thead class="ez-draft-conflict-list-header">
+    <tr>
+        <th class="ez-draft-conflict-list-head-version">Version</th>
+        <th class="ez-draft-conflict-list-head-language">Language</th>
+        <th class="ez-draft-conflict-list-head-author">Author</th>
+        <th class="ez-draft-conflict-list-head-saved">Last saved</th>
+    </tr>
+    </thead>
+    <tbody class="ez-draft-conflict-list-content">
+    {{#each drafts}}
+        <tr class="ez-draft-conflict-list-row">
+            <td class="ez-draft-conflict-list-cell">
+                {{versionNo}}
+            </td>
+            <td class="ez-draft-conflict-list-cell">
+                {{languageCodes}}
+            </td>
+            <td class="ez-draft-conflict-list-cell">
+                {{resources.Creator}}
+            </td>
+            <td class="ez-draft-conflict-list-cell ez-draft-conflict-list-cell-option">
+                {{modificationDate}}
+                <div class="ez-draft-conflict-list-row-options">
```

Comment:
in the initial specification, the button is supposed to be inside the table (this works better here since we only have one table) and that's actually easier to do, you just have to add a new column in the table and the `.ez-draft-conflict-link` elements are hidden (`visibility: hidden`) until the row is selected.